test/runtests.jl

```diff
@@ -4,6 +4,6 @@ using Images
 using LaTeXStrings
 using Test, ReferenceTests
 
+include("unit.jl")
 include("svg.jl")
 include("animations.jl")
```

Comment:
According to BlueStyle, these includes actually should be in alphabetical order.

Reply:
I don't think this applies for tests as I want to run unit tests first as that makes sense in my opinion.

Reply:
I agree with you on principal. However, it seems like this section here on [module imports]( https://github.com/invenia/BlueStyle#module-imports) and [test formatting](https://github.com/invenia/BlueStyle#test-formatting) are at odds somewhat.

Just keep the order you have unless if you think it should change based on BlueStyle are my thoughts.

Reply:
I didn't find specifics on that as they mention `using` where the order is irrelevant but for `include` it matters.